scripts/ci/publish_typescript_package.py

```
@@ -102,6 +104,13 @@ def _pack_package(
     expected_version: str,
     run_npm: RunNpm,
 ) -> PackageArtifact:
+    try:
+        readme = (package_directory / "README.md").read_text(encoding="utf-8").strip()
+    except OSError as error:
+        raise PublicationError("Package README.md could not be read") from error
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)